syft/distro/distro.go

```diff
@@ -33,7 +38,7 @@ func NewDistro(t Type, ver, like string) (Distro, error) {
 
 // Name provides a string repr of the distro
 func (d Distro) Name() string {
-	return string(d.Type)
+	return d.Type
```

Comment:
Should we add a `Name` field to `Distro`. We can then account for these three fields.
```
NAME
ID
ID_LIKE
```

Reply:
I can't think of a reason not to!

I was just refreshing myself on the [spec for os-release](https://www.linux.org/docs/man5/os-release.html). I don't know that we need to reflect _all_ fields, but this list of fields may be worth consulting as we thinking about the future of distro identification.